src/main/java/mate/hw13/sql/CrudOperations.java

```diff
@@ -0,0 +1,49 @@
+package mate.hw13.sql;
+
+import util.JdbcConnectionUtil;
+
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class CrudOperations {
```

Comment:
use separate repository class for each table

Reply:
I don't understand, can you explain please?